### PR TITLE
Merging the imgRef and asset into one, yay :) Issue #2528

### PR DESCRIPTION
**Pull request by joanofdart**
Merged both imgRef and asset directives in one.
Changed the **imgRef** naming convention to just **ref**
Removed **placeholderUrl** since we don't use it anymore and the images are already within each theme.

Issue #2528

### CODE REVIEW

File: apps/catalog/catalog/src/app/marketplace/title/view/view.component.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <article fxLayout="row" fxLayout.lt-sm="column" fxFlex>
       <!-- TODO#1763 get poster by countries from promotionalElements -->
       <div *ngIf="movie.promotionalElements.poster[0].media as poster">
-        <img class="movie-poster" [imgRef]="poster" [placeholderUrl]="'assets/images/movieposter-placeholder.png'" alt="movie poster"/>
+        <img class="movie-poster" [ref]="poster" asset="assets/images/movieposter-placeholder.png" alt="movie poster"/>
```
Comment (GrandSchtroumpf) [inline apps/catalog/catalog/src/app/marketplace/title/view/view.component.html:9]: I think you can do `asset="empty_poster.webp"` here. To verify if it works you need to run the catalog app : 
`ng serve catalog`

Comment (joanofdart) [reply apps/catalog/catalog/src/app/marketplace/title/view/view.component.html:9]: image

Works 🎉

File: libs/organization/src/lib/organization/components/card/card.component.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <mat-card fxLayout="column" fxLayoutAlign="center center">
-  <img mat-card-image [imgRef]="org.logo" placeholderAsset="Organization_250.png" alt="organization logo" />
+  <img mat-card-image [ref]="org.logo" asset="Organization_250.png" alt="organization logo" />
```
Comment (GrandSchtroumpf) [inline libs/organization/src/lib/organization/components/card/card.component.html:2]: can you change the asset to `asset="empty_organization.webp"` instead ? thanks.

File: libs/organization/src/lib/organization/layout/view/view.component.html [hunk 3]
```diff
@@ -15,7 +15,7 @@
           <span>{{ org.denomination.full }}</span>
           <span class="mat-caption">{{ org.activity | toLabel: 'orgActivity' | async }}</span>
         </div>
-        <img [imgRef]="org.logo" placeholderAsset="Organization_250.png" alt="organization logo" />
+        <img [ref]="org.logo" asset="Organization_250.png" alt="organization logo" />
```
Comment (GrandSchtroumpf) [inline libs/organization/src/lib/organization/layout/view/view.component.html:18]: same here.

File: libs/organization/src/lib/organization/pages/view/view.component.html [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <ng-container *ngIf="organization$ | async as org">
   <header fxLayout="row" fxLayoutAlign="space-between start">
     <article fxLayout="row" fxLayoutAlign="space-around start">
-      <img [imgRef]="org.logo" placeholderAsset="Organization_250.png" alt="organization logo" />
+      <img [ref]="org.logo" asset="Organization_250.png" alt="organization logo" />
```
Comment (GrandSchtroumpf) [inline libs/organization/src/lib/organization/pages/view/view.component.html:4]: same here

Comment (joanofdart) [reply libs/organization/src/lib/organization/pages/view/view.component.html:4]: Done

File: libs/organization/src/lib/organization/layout/view/view.component.html (hunk 3 above)
Comment (joanofdart) [reply libs/organization/src/lib/organization/layout/view/view.component.html:18]: Done

File: libs/organization/src/lib/organization/components/card/card.component.html (hunk 2 above)
Comment (joanofdart) [reply libs/organization/src/lib/organization/components/card/card.component.html:2]: Done

**Pull request merged**